src/application/worker/loop.js

```diff
@@ -69,8 +69,21 @@ function loop(delta, features, fftOutput) {
       location: linkLocation,
       args: linkArguments,
       min,
-      max
+      max,
+      source
     } = link;
+
+    const moduleId = store.state.inputs.inputs[inputId].data.moduleId;
+
+    if (
```

Comment:
Please add a comment to make clear that the links of a module are not updated when the module is not enabled?